plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/configrepo/contract/material/CRPluggableScmMaterial.java

```diff
@@ -22,13 +22,13 @@ public CRPluggableScmMaterial(String materialName,String scmId,String folder,Str
         super(TYPE_NAME,materialName);
         this.scm_id = scmId;
         this.destination = folder;
-        this.filter = new CRFilter(Arrays.asList(filters));
+        this.filter = new CRFilter(Arrays.asList(filters),false/*not supported in 16.6*/);
```

Comment:
You're right about that. Might need to spend some time thinking about how we can go about adding whitelist support in pluggable SCMs.